manifests/p/PhilipGerke/Odassey/1.1.7/PhilipGerke.Odassey.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PhilipGerke.Odassey
+PackageVersion: 1.1.7
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+- silentWithProgress
```

Comment:
btw, you should do interactive when you can click next and only silent when its a progress bar since NSIS only have /S

Reply:
I have always wondered why almost all NSIS installers are labeled silentWithProgress. Thanks for clarifying.